README.md

```diff
@@ -14,6 +14,10 @@ The architecture is supported by several managed services including [Amazon Cogn
 
 ![Retail Demo Store Architecture](./workshop/images/retaildemostore-architecture.png)
 
+## Site Optimization with Layer0
```

Comment:
I would prefer to see this in the Partner integration chapter to keep it consistent